python/lsst/analysis/drp/plotUtils.py

```diff
@@ -135,14 +142,21 @@ def addPlotInfo(fig, plotInfo):
     photocalibDataset = "None"
     astroDataset = "None"
 
-    plt.text(0.02, 0.98, "Run:" + plotInfo["run"], fontsize=8, alpha=0.8, transform=fig.transFigure)
-    datasetType = ("Datasets used: photocalib: " + photocalibDataset + ", astrometry: " + astroDataset)
+    plt.text(0.01, 0.98, "Plot Name: " + plotInfo["plotName"], fontsize=8, transform=fig.transFigure)
+    plt.text(0.01, 0.95, "Run: " + plotInfo["run"], fontsize=8, alpha=0.6, transform=fig.transFigure)
+    datasetType = ("Datasets Used: photocalib: " + photocalibDataset + ", astrometry: " + astroDataset)
     tableType = "Table: " + plotInfo["tractTableType"]
-    dataIdText = "Tract: " + str(plotInfo["tract"]) + " , Filter: " + plotInfo["filter"]
-    plt.text(0.02, 0.95, datasetType, fontsize=8, alpha=0.8, transform=fig.transFigure)
-
-    plt.text(0.02, 0.92, tableType, fontsize=8, alpha=0.8, transform=fig.transFigure)
-    plt.text(0.02, 0.89, dataIdText, fontsize=8, alpha=0.8, transform=fig.transFigure)
+    tractText = "Tract: " + str(plotInfo["tract"])
+    bandsText = "Bands: " + plotInfo["bands"]
+    SNText = "S/N: " + str(plotInfo["SN"])
+    visitText = "Visit: " + str(plotInfo["visit"])
+
+    plt.text(0.01, 0.92, datasetType, fontsize=8, alpha=0.6, transform=fig.transFigure)
+    plt.text(0.01, 0.89, tableType, fontsize=8, alpha=0.6, transform=fig.transFigure)
+    plt.text(0.01, 0.86, tractText, fontsize=8, alpha=0.6, transform=fig.transFigure)
+    plt.text(0.01, 0.83, visitText, fontsize=8, alpha=0.6, transform=fig.transFigure)
+    plt.text(0.01, 0.80, bandsText, fontsize=8, alpha=0.6, transform=fig.transFigure)
+    plt.text(0.01, 0.77, SNText, fontsize=8, alpha=0.6, transform=fig.transFigure)
```

Comment:
I don't think you can get away with so many printed lines for this method. More often than not, the information overlaps the plotting region. From my testing, I think you can guarantee no more than 4 lines worth of information can be squeezed in at the top of the figure.

Reply:
I've rewritten the `addPlotInfo` method to look like this:
```
def addPlotInfo(fig, plotInfo):
    """Add useful information to the plot

    Parameters
    ----------
    fig : `matplotlib.figure.Figure`
    plotInfo : `dict`

    Returns
    -------
    fig : `matplotlib.figure.Figure`
    """

    # TO DO: figure out how to get this information
    photoCalibDataset = "None"
    astrometryDataset = "None"

    plotName = f"{plotInfo['plotName']}"
    datasetType = f"\nRun: {plotInfo['run']}"
    datasetsUsed = f"\nPhotoCalib: {photoCalibDataset}, Astrometry: {astrometryDataset}"
    tableType = f"\nTable: {plotInfo['tractTableType']}"
    if str(plotInfo["tract"]) != "N/A":
        tractText = f", Tract: {str(plotInfo['tract'])}"
    else:
        tractText = ""
    if str(plotInfo["visit"]) != "N/A":
        visitText = f", Visit: {str(plotInfo['visit'])}"
    else:
        visitText = ""
    bandsText = f", Bands: {''.join(plotInfo['bands'].split(' '))}"
    SNText = f", S/N: {str(plotInfo['SN'])}"

    infoText = f"{datasetType}{datasetsUsed}{tableType}{tractText}{visitText}{bandsText}{SNText}"

    plt.text(0.01, 0.99, plotName, fontsize=8, transform=fig.transFigure, ha='left', va='top')
    plt.text(0.01, 0.99, infoText, fontsize=8, transform=fig.transFigure, ha='left', va='top', alpha=0.6)

    return fig
```

Reply:
This has a number of advantages, including not having to specify the position of each new line in the printed information (using '\n' instead). Furthermore, I've added some logic that only prints one of Tract and Visit, as printing them both does not make sense.